symphony/runner/simbricks/runner/main_runner/plugins/plugin.py

```
@@ -9,7 +9,7 @@ class FragmentRunnerPlugin(abc.ABC):
     @staticmethod
     @abc.abstractmethod
     def name() -> str:
-        pass
+        pass # pytype: disable=bad-return-type
```

Please raise an Exception here. I think this method is callable as it is static.